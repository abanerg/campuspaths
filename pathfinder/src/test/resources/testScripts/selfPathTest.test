# Create graph

CreateGraph graph

AddNode graph A

AddEdge graph A A 10.000

# Should be path from A to A with cost 0.000 despite edge being added

FindPath graph A A
