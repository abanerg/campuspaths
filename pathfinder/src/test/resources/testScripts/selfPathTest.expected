# Create graph

created graph graph

added node A to graph

added edge 10.000 from A to A in graph

# Should be path from A to A with cost 0.000 despite edge being added

path from A to A:
total cost: 0.000

